replication/README.md

```diff
@@ -301,6 +302,9 @@ message ResyncVolumeResponse{
   // The default value is false.
   // This field is REQUIRED.
   bool ready = 1;
+  // Timestamp of last resync update on the volume.
+  // This field is optional.
+  .google.protobuf.Timestamp updated_time = 2;
```

Comment:
Why `updated_time`, and not `last_resync_time` or something descriptive? A status output in a CO would probably also have a _updated time_ which might be different from the last completed(?) resync.

Reply:
I had thought about using last_sync_update https://github.com/csi-addons/spec/issues/24 but the Response struct name itself has ResyncVolumeResponse added choosing this as updated time so that not to repeat the same working of resync in the field also. and the CO can choose to add this to the field it wants it does need not use the variable name we are using.